runelite-client/src/main/java/net/runelite/client/plugins/combatcounter/CombatCounter.java

```diff
@@ -316,7 +320,7 @@ public void onAnimationChanged(AnimationChanged event)
 								for (NPC nearby : this.client.getNpcs())
 								{
 									int distance = npc.getWorldLocation().distanceTo(nearby.getWorldLocation());
-									if (distance <= 1 && npc != nearby)
+									if (distance <= 1 && npc.equals(nearby))
```

Comment:
Is this change on purpose (!= would work here, so would == but I don't know if youre supposed to be checking equality or inequality)